src/sv-pipeline/04_variant_resolution/scripts/clean_vcf_part5_update_records.py

```diff
@@ -160,8 +164,9 @@ def main():
                 if record.id in sexchr_revise:
                     for sample in record.samples:
                         if sample in male_samples:
-                            cn = int(record.samples[sample]['RD_CN'])
-                            if cn is not None and cn > 0:
+                            cn = record.samples[sample]['RD_CN']
+                            if cn is not None and int(cn) > 0:
+                                cn = int(cn)
```

Comment:
Are you separating `int` casting because you expect `record.samples[sample]['RD_CN']` containing values not convertible to the `int` type?

Reply:
Yes it could be `None` which causes an error. Is there a better way?